deluge/common.py

```diff
@@ -720,6 +720,23 @@ def parse_human_size(size):
     raise InvalidSize(msg % (size, tokens))
 
 
+def wrap_url_with_html(text):
+    """
+    A simple regex sub to wrap all occurrences of URLs with HTML
```

Comment:
Try to avoid describing the implementation detail, using regex, in docstring/comment. Also 'simple' is quite subjective so should be avoided.